services/horizon/internal/app.go

```diff
@@ -48,6 +48,7 @@ type App struct {
 	ingester                     *ingest.System
 	reaper                       *reap.System
 	ticks                        *time.Ticker
+	WhiteListAccount             string
```

Comment:
- i don't think this member needs to be public. make it private
- add a smal comment explaining what this is and why is it here (for caching the whitelist address), regardless that the other members here are undocumented.

Reply:
moved to Config